controller/raft/mesh/peerconn.go

```diff
@@ -0,0 +1,159 @@
+package mesh
+
+import (
+	"github.com/openziti/channel"
+	"github.com/openziti/foundation/util/concurrenz"
+	"github.com/pkg/errors"
+	"github.com/sirupsen/logrus"
+	"io"
+	"net"
+	"time"
+)
+
+func newRaftPeerConn(peer *Peer, localAddr net.Addr) *raftPeerConn {
+	return &raftPeerConn{
+		peer:        peer,
+		localAddr:   localAddr,
+		readTimeout: newDeadline(),
+		readC:       make(chan []byte, 16),
+		closeNotify: make(chan struct{}),
+	}
+}
+
+type raftPeerConn struct {
```

Comment:
I have been mostly asking for doc on major interfaces, but this seems important. Consider doc?

Reply:
Fixed